python/SoapySDR.in.i

```diff
@@ -158,7 +175,6 @@ if os.name == 'nt' and hasattr(os, 'add_dll_directory'):
 %ignore SoapySDRArgInfoList_clear;
 %include <SoapySDR/Types.h>
 %include <SoapySDR/Errors.h>
-%include <SoapySDR/Version.h>
```

Comment:
this breaks the CI

Reply:
Hm, it works for me (TM). The SWIG version Travis uses is pretty old, so it may be a change in behavior. I'll put this back and repush.

Reply:
I tried it on 20.04, same thing, the .h files pull in the #define constants all uppercase not the same as the .hpp for the c++ namespaced functions

Reply:
Odd, not sure why that breaks anything. It should've processed Version.hpp already. Oh well, fixing.